firmware/.gitignore

```diff
@@ -0,0 +1,15 @@
+#ignore files that may contain sensitive client information
+SetupFiles/
+setupFirmware.h
+
+#ignore vscode settings
+.vscode/
+
+#ignore Argon binary files
```

Comment:
Can we also ignore the binaries for the tests? Like `firmware/test/ConsoleTests`?